app/views/listings/confirm_match.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="title is-3">Confirm Match</div>
+
+<%= link_to("Autogenerate system email to offer owner", root_path, class: "button is-primary is-outlined") %>
+<br>
+<%= link_to("Autogenerate system text to offer owner", root_path, class: "button is-primary is-outlined") %>
+<br>
+<%= link_to("Log email to offer owner", new_communication_log_path, class: "button is-primary is-outlined") %>
+<br>
+<%= link_to("Log text to offer owner", new_communication_log_path, class: "button is-primary is-outlined") %>
+<br>
+<%= link_to("Cancel", root_path, class: "button is-primary is-outlined") %>
+<br>
```

Comment:
Still need to decide options for Confirm, depending on user and instance settings. This is placeholder for now.